README.md

```diff
@@ -44,10 +44,27 @@ if (isDev) {
 
 ## Devtron API
 
-### `await devtron.install()`
+### `await devtron.install(options)`
 
 Installs Devtron into the Electron app. Refer to [Configuring an Electron App to use Devtron](#configuring-an-electron-app-to-use-devtron) for installation instructions.
 
+#### `Options`
+
+| Option       | Type                                            | Default     | Description                                                                                                                           |
+| ------------ | ----------------------------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| `quiet`      | `boolean`                                       | `false`     | Suppresses all logs except `error` level logs from appearing in the terminal. Precisely, it ignores `debug`, `info`, and `warn` logs. |
+| `ignoreLogs` | `Array<'debug' \| 'info' \| 'warn' \| 'error'>` | `undefined` | List of log levels to ignore, e.g. `['debug', 'info']`. Overrides the `quiet` option if both are set.                                 |
+
+Examples:
+
+```js
+await devtron.install({ quiet: true });
+```
+
+```js
+await devtron.install({ ignoreLogs: ['debug', 'info'] });
+```
```

Comment:
Agreed with @erickzhao about a `logLevel` option. Then these two examples could be
```
devtron.install({ logLevel: [] })
devtron.install({ logLevel: ['info', 'warn'] })
``` 

I'm partial to the idea of the `quiet` option as a shorthand though. Open to either keeping it or removing it.